mtp_cashbook/templates/core/index.html

```diff
@@ -1,15 +1,22 @@
 {% extends "base.html" %}
 
-{% block title %}Yo{% endblock title %}
-
 {% block content %}
-<div class="row">
-  <div class="small-12 columns moj-wrapper">
-    <h1>
-      Yo MoJ222!&nbsp;
-      <a href="//github.com/mixxorz/generator-moj-django/"><i class="fa fa-github"></i></a>
-    </h1>
-    <p>Welcome to your MoJ Django app.</p>
-  </div>
-</div>
+  <h1>Money sent to prisoners</h1>
+
+  <ul class="SummaryBoxes">
+    <li class="SummaryBoxes--fresh">
```

Comment:
And the `--fresh` bit is my stupid name for the outstanding payments variation of the summary boxes. Maybe `--unprocessed` would be a better variation classname?